### PR TITLE
docs: Minor tweaks to the Grafana Fundamentals tutorial

### PR DESCRIPTION
**Pull request by tomglenn**
Updated the Grafana Fundamentals tutorial to fix reference to incorrect section for annotations, as well as added a note around the example alert potentially being fired due to the `/metrics` endpoint accessed by Grafana.

### CODE REVIEW

**Review by mdcruz — APPROVED** [review]
LGTM! 🛳️

**Review by tonypowa — APPROVED** [review]
LGTM @tomglenn !

**Comment by moxious** [comment]
@imatwawana could you have a look and see if there are any appropriate backport labels?   Reviewed from our team side, just needs final check & merge - this is @tomglenn 's first docs PR (tada!)

**Comment by imatwawana** [comment]
> @imatwawana could you have a look and see if there are any appropriate backport labels? Reviewed from our team side, just needs final check & merge - this is @tomglenn 's first docs PR (tada!)

Congrats @tomglenn! And no backport labels needed for tutorials, no.

**Review by imatwawana — APPROVED** [review]

File: docs/sources/tutorials/grafana-fundamentals/index.md [hunk 1]
```diff
@@ -357,6 +358,10 @@ We have now configured an alert rule and a contact point. Now let's see if we ca
 
 Once the query `sum(rate(tns_request_duration_seconds_count[5m])) by(route)` returns a value greater than `0.2` Grafana will trigger our alert. Browse to the Request Bin we created earlier and find the sent Grafana alert notification with details and metadata.
 
+{{< admonition type="note"
```
Comment (bw-dani) [inline docs/sources/tutorials/grafana-fundamentals/index.md:361]:
```suggestion
{{< admonition type="note" >}}
```
Could be this?

Comment (imatwawana) [reply docs/sources/tutorials/grafana-fundamentals/index.md:361]: Ah that was all me!! That's probably it!

**Pull request merged**